openfisca_core/commons/tests/test_rates.py

```diff
@@ -0,0 +1,28 @@
+"""Tests for the rates module."""
+
+import numpy
+from numpy.testing import assert_array_equal
+
+from openfisca_core import commons
+
+
+def test_average_rate_when_varying_is_zero():
+    """When the varying gross income crosses zero it yields infinity."""
+
+    target = numpy.array([1, 2, 3])
+    varying = 0
+
+    result = commons.average_rate(target, varying)
+
+    assert_array_equal(result, [- numpy.inf, - numpy.inf, - numpy.inf])
+
+
+def test_marginal_rate_when_varying_is_zero():
+    """When the varying gross income crosses zero it yields infinity."""
```

Comment:
Maybe add more details to show a distinction from the the previous test? 
> ```suggestion
>     """Yields infinity when the varying gross income crosses zero."""
> ```